docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -589,6 +589,17 @@ table in the metastore, using its existing transaction logs and data files::
 To prevent unauthorized users from accessing data, this procedure is disabled by default.
 The procedure is enabled only when ``delta.register-table-procedure.enabled`` is set to ``true``.
 
+.. _delta-lake-unregister-table:
+
+Unregister table
+^^^^^^^^^^^^^^^^
+The connector can unregister existing Delta Lake tables from the catalog.
```

Comment:
"Catalog" is a copy from the Iceberg documentation, I would imagine. Metastore is more Delta Lake appropriate.

```suggestion
The connector can unregister existing Delta Lake tables from the metastore.
```